### PR TITLE
ACQ-741 Update the AI PL overview section to the page

### PR DESCRIPTION
**Pull request by dmantonyuk**
We are going to convert the top section of the AI PL page to better overview the series and we will update it as things launch. 

## Links
https://codedotorg.atlassian.net/browse/ACQ-741
https://code.org/ai/pl/101
https://www.figma.com/file/hNBxdgZ40KH4UZFG7ZEVo8/AI-Landing-Page-Updates---Spring-2023?type=design&node-id=2774-7991&mode=design

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
image

File: pegasus/sites.v3/code.org/public/ai/pl/101/index.haml [hunk 1]
```diff
@@ -30,14 +30,30 @@ theme: responsive_full_width
...
-
-= view :section_divider_line
+- video_101_index = 5
+- video_101_tags = [hoc_s(:module_label_new), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon)]
 
-%section.newsletter-signup
+%section.bg-neutral-light.video-101.bg-neutral-light{id: "ai-pl-callout"}
+  .wrapper
+    .text-wrapper
+      %h2.heading-lg=hoc_s(:ai_pl_callout_title)
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/pl/101/index.haml]: This doesn't need the `.heading-lg` class.

File: pegasus/sites.v3/code.org/public/ai/pl/101/index.haml [hunk 2]
```diff
@@ -30,14 +30,30 @@ theme: responsive_full_width
...
+      %p.video-101__text=hoc_s(:ai_pl_callout_desc_01)
+
+    - video_101_index.times do |index|
+      .action-block.action-block--two-col
+        .video-101__image
+          %img{src: "/images/ai/pl/101/coming-soon.jpg", alt: hoc_s("ai_pl_101_video_title_#{index + 1}"), class: "col-50"}
+        .text-wrapper.col-50
+          %p.overline=video_101_tags[index]
+          %h2=hoc_s("ai_pl_101_video_title_#{index + 1}")
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/pl/101/index.haml]: This should be an `h3`.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -448,7 +451,7 @@
   teachai_launch_blog_desc_02: "TeachAI aims to integrate AI education into primary and secondary curricula around the world through new reports, policy recommendations, and public engagement opportunities."
 
   ai_pl_callout_title: "AI 101 for Teachers"
-  ai_pl_callout_desc_01: "Discover the groundbreaking world of artificial intelligence (AI) and its transformative potential in education with our comprehensive online learning series for teachers."
+  ai_pl_callout_desc_01: "This is a free, foundational online learning series for any teacher and educator interested in the groundbreaking world of artificial intelligence (AI) and its transformative potential in education. Partners Code.org, ETS, ISTE and Khan Academyare offering engaging sessions with renowned experts that will demystify AI, explore responsible implementation, address bias, and showcase how AI-powered learning can revolutionize student outcomes. Join us on this journey of exploration and empowerment, and unlock the future of teaching with and about AI."
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Missing a space between `Academy` and `are` here: `Khan Academyare offering`.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 4]
```diff
@@ -532,6 +535,16 @@
...
+  ai_pl_101_video_desc_2: "Introduction to the fundamentals of Artificial Intelligence which will equip you with the foundational knowledge to understand the profound impact of AI in our society and its increasing relevance to the field of education. You will gain practical insights into leveraging AI for tasks in the classroom, and saving time on administrative tasks."
+  ai_pl_101_video_title_3: "Transforming Learning with AI"
+  ai_pl_101_video_desc_3: "Explore the intersection of AI and education. This session explores how AI can be combined with pedagogy to enhance student learning. It is a must-attend for educators seeking to leverage AI to transform their teaching practices to enrich their students’ learning experiences."
+  ai_pl_101_video_title_4: "Ensuring a Responsible Approach to AI"
+  ai_pl_101_video_desc_4: "Explore the ethical considerations and best practices for incorporating artificial intelligence in educational environments.  This session will provide concrete strategies, resources, and real-world examples, all aimed at understanding the potential biases of AI, and the ethical considerations that come with it."
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Extra space here: `environments.  This`.

**Review by kelbyhawn — APPROVED** [review]
Looks great! 🙌

File: pegasus/sites.v3/code.org/public/css/ai-pl-101-styles.scss [hunk 5]
```diff
@@ -180,3 +181,43 @@
...
+}
+
+.video-101 {
+  padding-bottom: 8px;
+
+  &__text {
+    margin-bottom: 32px;
+  }
+
+  & a.link-button {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/ai-pl-101-styles.scss:197]: @dmantonyuk I didn't catch this before, but you can add [this existing `secondary` class](https://github.com/code-dot-org/code-dot-org/blob/34175761b482c6a4df605c1823389a04e431d79c/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L226-L233) to these buttons in the haml file and don't need to duplicate the styles here.

**Pull request merged**